packages/kcmsf/src/pages/matchList.tsx

```diff
@@ -98,23 +98,20 @@ export const MatchList = () => {
                       </Table.Td>
                       <Table.Td>
                         <Text fw={700} miw={200} ta={"start"}>
-                          {match.teams.left.teamName}
+                          {match.left.teamName}
                         </Text>
                       </Table.Td>
                       <Table.Td>
                         <Text fw={700} miw={200} ta={"start"}>
-                          {match.teams.right.teamName}
+                          {match.right.teamName}
                         </Text>
                       </Table.Td>
                       <Table.Td>
                         <Center>
                           <MatchStatusButton
                             status={match.results ? "end" : "future"}
                             id={match.id}
-                            teams={match.teams}
-                            matchType={
-                              match.matchType == "primary" ? "pre" : "main"
-                            }
+                            matchType={match.matchType == "pre" ? "pre" : "pre"}
```

Comment:
```suggestion
                            matchType={match.matchType}
```
良く分かりませんが多分これでいいのでは...？